<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RectRubberWidget</class>
 <widget class="QWidget" name="RectRubberWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RectRubberWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="l_ar">
       <property name="text">
        <string>ar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="le_ar"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="l_br">
       <property name="text">
        <string>br</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="le_br"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="l_R">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="le_R"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QCheckBox" name="chb_manual">
       <property name="text">
        <string>Задать самому</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="l_k">
       <property name="text">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="le_k">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="gb_Whole">
     <property name="title">
      <string>Целая</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="l_Dro">
          <property name="text">
           <string>Dr</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_Dro"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="l_ako">
          <property name="text">
           <string>ak</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_ako"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="b_Calco">
        <property name="text">
         <string>Расчёт</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_bko">
        <property name="text">
         <string>bk = ?</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_Ako">
        <property name="text">
         <string>Ak = ?</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="l_Bko">
        <property name="text">
         <string>Bk = ?</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="l_Hko">
        <property name="text">
         <string>Hk = ?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="gb_Cut">
     <property name="title">
      <string>Разрезанная</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="l_ak">
            <property name="text">
             <string>ak</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_ak"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="l_bk">
            <property name="text">
             <string>bk</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_bk"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="b_Calc">
          <property name="text">
           <string>Расчёт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_Ak">
          <property name="text">
           <string>Ak = ?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_Bk">
          <property name="text">
           <string>Bk = ?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_Hk">
          <property name="text">
           <string>Hk = ?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_lr">
          <property name="text">
           <string>lr = ?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_Dr">
          <property name="text">
           <string>Dr = ?</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>chb_manual</sender>
   <signal>clicked(bool)</signal>
   <receiver>le_k</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_ar</sender>
   <signal>textChanged(QString)</signal>
   <receiver>le_br</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
